src/jobs/config_sync_job.ts

```diff
@@ -123,6 +222,19 @@ export class ConfigSyncJob extends SyncJob {
 
     // do not delete now, set markedToDelete to now and delete after some days to allow users to set time to completed tasks which are not fetched from primary etc.
     for (const mapping of this._connection.mappings) {
+      // delete object from secondary service because it was deleted in the primary
+      const primaryObjectExists = objectsToSync.find((obj: ServiceObject) => {
+        return obj.id === mapping.primaryObjectId
+      })
+      if (!primaryObjectExists) {
+        //delete secondary object in the service
+        const secondaryServiceObject = mapping.mappingsObjects[0].service === "Toggl Track" ? mapping.mappingsObjects[0] : mapping.mappingsObjects[1]
+        //console.log(`about to delete obj from ${secondaryServiceObject.service} with Id ${secondaryServiceObject.id} and name ${secondaryServiceObject.name}`)
+
+        //We do not want to delete immediately, it will be deleted later
+        //await syncedService.deleteServiceObject(secondaryServiceObject.id, 'tag')
+      }
+
```

Comment:
Tohle je kvůli čemu? Přidání issue info pro TE když je mažu z toggl?

Reply:
Když je osdtraněn zdrojový objekt, tak chceš mazat až po nějaké době, ale tohle taky nechápu. Já jsem to neměnil, jenom jsem to zabalil do funkce

Reply:
Prosím tedy o prozkoumání a zjištění proč je zde explicitní vazba na Toggl. Případně prosím o další úklid

Reply:
Explicitní vazba na Toggl je tady proto, že z projektových nástrojů nechceš objekty mazat. To je povolené pouze u timer nástrojů, takže výhradně pro Toggl. S dalším rozšířením aplikace by rozhodně dávalo smysl přidat nástroji atribut, jakého je typu a to používat místo porovnávání se stringem.